omisego-sdk/src/main/java/co/omisego/omisego/websocket/listener/SocketCustomEventListener.kt

```diff
@@ -32,32 +32,32 @@ interface SocketCustomEventListener {
         }
 
         /**
-         * A convenient method for listening for any events, but the events will be filtered out by the provided `FilterStrategy`
+         * A convenient method for listening for the specific topic.
          *
-         * @param listener A [SocketCustomEventListener] implementation
-         * @param strategy A [FilterStrategy] that used for filtering an event.
+         * @param listenable A listenable object that provide the `SocketTopic`.
+         * @param lambda A lambda which receives the `SocketEvent` object regarding the topic.
          */
-        fun forEvents(
-            strategy: FilterStrategy,
-            listener: SocketCustomEventListener
+        inline fun forTopic(
```

Comment:
should we rename this one  `forListenable` version and a `forTopic` version that'll take a topic as param? not sure it's necessary, but the name could lead to confusion?